src/main/java/com/engdev/account/AccountApp.java

```diff
@@ -0,0 +1,11 @@
+package com.engdev.account;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class AccountApp {
```

Comment:
The class should simply be called application

Reply:
Ok, let me chage it